### PR TITLE
Cross-compile to netcore50 to support EF on UWP

### PR DESCRIPTION
**Pull request by natemcmaster**
Part of https://github.com/aspnet/EntityFramework/pull/5021

### CODE REVIEW

**Comment by davidfowl** [comment]
Why is there a runtimes section?

**Comment by natemcmaster** [comment]
@davidfowl it was a workaround for a bug in dotnet-pack for a previous draft of this PR and I forgot to check if I could remove it. It appears this works without so i'll take it out.

File: src/Microsoft.Data.Sqlite/project.json [hunk 1]
```diff
@@ -27,6 +27,31 @@
...
+        "System.Reflection.Extensions": "4.0.0",
+        "System.Resources.ResourceManager": "4.0.0",
+        "System.Runtime.Extensions": "4.0.10",
+        "System.Runtime.InteropServices": "4.0.20",
+        "System.Text.Encoding": "4.0.10",
+        "System.Threading": {
+          "type": "build",
+          "version": "4.0.10"
+        },
+        "System.Text.RegularExpressions": {
```
Comment (davidfowl) [inline src/Microsoft.Data.Sqlite/project.json]: Why would you need build time dependencies?

Comment (natemcmaster) [inline src/Microsoft.Data.Sqlite/project.json]: This, and all other build time dependencies I added, are only there so restore will pass.

**Comment by natemcmaster** [comment]
@davidfowl we have ask-mode permission to merge. Even though we all agree -- cross-compiling is an unfortunate workaround -- is this change okay to merge?

**Pull request merged**